torch/_inductor/codecache.py

```diff
@@ -1638,6 +1641,7 @@ def compile(
             "wrapper.cpp",
             extra=cpp_command,
             specified_dir=specified_output_path,
+            key=model_name,
```

Comment:
Can we just use `config.aot_inductor.model_name_for_generated_file` here? Then we won't need to update `compile_fx.py`.

Reply:
@desertfire Fixed!